tests/installation/boot_encrypt.pm

```diff
@@ -17,13 +17,15 @@ use utils;
 use testapi qw/get_var record_soft_failure/;
 
 sub run() {
-    if (get_var('ENCRYPT_ACTIVATE_EXISTING') && !get_var('ENCRYPT_FORCE_RECOMPUTE')) {
-        record_soft_failure('bsc#993247 fate#321208: activating existing encrypted volume does *not* yield an encrypted system if not forcing');
-        return;
+    if (
+        get_var('ENCRYPT_ACTIVATE_EXISTING') && !get_var('ENCRYPT_FORCE_RECOMPUTE') and (sle_version_at_least('12-SP4') or sle_version_at_least('13'))
```

Comment:
that should not make any difference here because it is still a logical end, precedence does not matter. Anyway, updated.